ctf/src/systems/on_leave/update_gamemode.rs

```diff
@@ -35,9 +35,9 @@ impl<'a> System<'a> for UpdateGameModeOnPlayerLeave {
 			let team = data.teams.get(*ent).unwrap();
 
 			if *team == RED_TEAM {
-				data.gamemode.redteam -= 1;
+				data.gamemode.redteam = std::cmp::max(data.gamemode.redteam - 1, 0);
 			} else if *team == BLUE_TEAM {
-				data.gamemode.blueteam -= 1;
+				data.gamemode.blueteam = std::cmp::max(data.gamemode.blueteam - 1, 0);
```

Comment:
This doesn't solve the bug. If the subtraction would cause the variable to underflow then the `data.gamemode.redteam - 1` would underflow (and panic) anyway before doing the comparison.

I think this does what you want
```suggestion
				data.gamemode.blueteam -= std::cmp::min(data.gamemode.blueteam, 1);
```